inc/class-sitemaps.php

```diff
@@ -1223,18 +1223,20 @@ function output() {
 	function sitemap_url( $url ) {
 
 		// Create a DateTime object date in the correct timezone
-		if ( isset( $url['mod'] ) ) {
-			$date = new DateTime( $url['mod'], new DateTimeZone( $this->get_timezone_string() ) );
-		}
-		else {
-			$date = new DateTime( date( 'y-m-d H:i:s' ), new DateTimeZone( $this->get_timezone_string() ) );
+		$date = '';
```

Comment:
I would put line 1226-1232 in a separate method, because this method is getting to much responsibilities.